tla-io/src/test/scala/at/forsyte/apalache/io/json/TestUJsonToTla.scala

```diff
@@ -67,67 +79,34 @@ class TestUJsonToTla extends AnyFunSuite with Checkers {
   }
 
   test("Predef sets (see #1281)") {
-    val sets: Seq[ValEx] = Seq(
-        TlaIntSet,
-        TlaNatSet,
-        TlaBoolSet,
-        TlaStrSet,
-    ).map { v =>
-      ValEx(v).withTag(Untyped)
-    }
+    val sets: Seq[TlaEx] = Seq(
+        tla.intSet(),
+        tla.natSet(),
+        tla.booleanSet(),
+        tla.stringSet(),
+    )
 
     sets.foreach { s =>
       assert(dec.asTlaEx(enc(s)) == s)
     }
 
   }
 
-  test("TypeReader correctly reads valid type strings and fails on invalid type strings") {
-    val valid: Seq[String] = Seq(
-        "Untyped",
-        "Int",
-        "Set(Bool)",
-        "() => a -> b",
-        "(UT, Int) => [x: Str]",
-    )
-
-    val invalid: Seq[String] = Seq(
-        "",
-        "SomeType",
-        "Untyped()",
-        "-12",
-        "Set(true)",
-        "() > a -> b",
-        "(UT, Int) => [x: 9]",
-        "Typed[Str](\"cake\")",
-    )
-
-    // No throw
-    valid.foreach { s =>
-      DefaultTagReader.apply(s)
-    }
-    invalid.foreach { s =>
-      assertThrows[JsonDeserializationError] {
-        DefaultTagReader.apply(s)
-      }
-    }
-
-  }
-
-  test("Deserializing a serialized IR produces an equivalent IR") {
-    val gens: IrGenerators = new IrGenerators {
-      override val maxArgs: Int = 3
-    }
-    val operators =
-      gens.simpleOperators ++ gens.logicOperators ++ gens.arithOperators ++ gens.setOperators ++ gens.functionOperators ++ gens.actionOperators ++ gens.temporalOperators
-    val genDecl = gens.genTlaDeclButNotVar(gens.genTlaEx(operators)) _
-    val prop = forAll(gens.genTlaModuleWith(genDecl)) { module =>
-      val moduleJson = enc(module)
-      val moduleFromJson = dec.asTlaModule(moduleJson)
-
-      module =? moduleFromJson
-    }
-    check(prop, minSuccessful(500), sizeRange(7))
-  }
+  // TODO: Uncomment once generated IRs are well-typed
+//  test("Deserializing a serialized IR produces an equivalent IR") {
+//    val gens: IrGenerators = new IrGenerators {
+//      override val maxArgs: Int = 3
+//    }
+//    val operators =
+//      gens.simpleOperators ++ gens.logicOperators ++ gens.arithOperators ++ gens.setOperators ++ gens.functionOperators ++ gens.actionOperators ++ gens.temporalOperators
+//    val genDecl = gens.genTlaDeclButNotVar(gens.genTlaEx(operators)) _
+//    val prop = forAll(gens.genTlaModuleWith(genDecl)) { module =>
+//      val moduleJson = enc(module)
+//      val moduleFromJson = dec.asTlaModule(moduleJson)
+//
+//      module =? moduleFromJson
+//    }
+//    check(prop, minSuccessful(500), sizeRange(7))
+//  }
```

Comment:
I think there is a better way to disable a test, e.g., by using `ignore` instead of `test`.

Reply:
> LGTM

Don't approve just yet, this is blocked by the tuple-based quantifier methods implementation.